.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,23 @@
-# Change Type
+## Change Type (Feature, Chore, Bug Fix, RFC)
```

Comment:
It seems @ceejbot is calling RFC's one-pagers, or are these two different things? Lines feel a bit blurred